src/scss/_variables.scss

```diff
@@ -68,3 +75,91 @@ $spacers: (
 $embed-responsive-aspect-ratios: ((16 9), (2 1));
 $enable-responsive-font-sizes: true;
 $rfs-factor: 4;
+
+//Global typography
```

Comment:
We already have typography in the design system taken from martins design guide? is this really needed ontop of that?